doc/ref_cert/RC2/index.rst

```diff
@@ -1,56 +1,10 @@
-Reference Conformance - Kubernetes based platform for Telecom
-=============================================================
-
-This is the Kubernetes Based Reference Conformance (RC-2)
-
-Release Information
--------------------
-
-**Bundle: 7**
-
-**Version: 0**
-
-**Release Date: 4th January 2022**
-
-Bundle/Version History
-----------------------
-
-============== ============= ===================
-Bundle.Version Date          Note
-============== ============= ===================
-0.0            15th May 2020 First Initial Draft
-4.0            25th Sep 2020 Baraque Release
-5.0            29th Jan 2021 Elbrus Release
-6.0            1st Jul 2021  Kali Release
-7.0            4th Jan 2022  Lakelse Release
-============== ============= ===================
-
-Overall Status
---------------
-
-+-----------------------------------+-----------------------------------+
-| Chapter                           | Status                            |
-+===================================+===================================+
-| Chapter 01 - Introduction         | Lots of SME feedback              |
-+-----------------------------------+-----------------------------------+
-| Chapter 02 - Kubernetes Test      | Lots of SME feedback              |
-| Cases and Requirements            |                                   |
-| Traceability                      |                                   |
-+-----------------------------------+-----------------------------------+
-| Chapter 03 - Kubernetes Testing   | Dickering over fine points        |
-| Cookbook                          |                                   |
-+-----------------------------------+-----------------------------------+
-| Chapter 04 - CNF Test Cases and   | Initial Framework Only            |
-| Requirements Traceability         |                                   |
-+-----------------------------------+-----------------------------------+
-| Chapter 05 - CNF Testing Cookbook | Initial Framework Only            |
-+-----------------------------------+-----------------------------------+
-| Chapter 06 - Gap analysis and     | Initial Framework Only            |
-| Development                       |                                   |
-+-----------------------------------+-----------------------------------+
-
-.. toctree::
-   :glob:
-   :maxdepth: 1
-
-   chapters/*
+=================================================
+Anuket Reference Conformance for Kubernetes (RC2)
+=================================================
+
+.. include:: chapters/chapter01.rst
+.. include:: chapters/chapter02.rst
+.. include:: chapters/chapter03.rst
+.. include:: chapters/chapter04.rst
+.. include:: chapters/chapter05.rst
+.. include:: chapters/chapter06.rst
```

Comment:
I am generally ok with putting everything onto a single page. However, does this concept also support header numbering which is important in the RM and RAs?

Reply:
yes it works.